sonic_api.yaml

```diff
@@ -294,11 +289,10 @@ paths:
           required: true
           type: string
           description: BGP profile name
-        - name: community_id
-          in: path
+        - name: profile
```

Comment:
`profile` is already present in the path on line 287. We need `community_id` as a parameter.

Reply:
It will be, it is in the body. Profile name is present in the path, but the profile contents aren't. I made an object (see below) called BgpProfile which has the community id information so that it serializes like this 
{"community_id":"string"}

Reply:
In that case, `name: profile` should be `name: BgpProfile` right?